src/Rule/VersionaddedDirectiveShouldHaveVersion.php

```diff
@@ -24,13 +24,10 @@
 use App\Value\ViolationInterface;
 use Composer\Semver\VersionParser;
 
-/**
- * @Description("Ensure a versionadded directive has a version which follows SemVer.")
- *
- * @ValidExample(".. versionadded:: 3.4")
- *
- * @InvalidExample({".. versionadded::", ".. versionadded:: foo-bar"})
- */
+#[Description('Ensure a versionadded directive has a version which follows SemVer.')]
+#[ValidExample('.. versionadded:: 3.4')]
+#[InvalidExample('.. versionadded::')]
+#[InvalidExample('.. versionadded:: foo-bar')]
```

Comment:
We don't have a lot of ValidExample/InvlidExample, so i prefer repeatable attribute type hinted `string` instead `string|string[]`

(It also simplify a little bit md doc generation)

Reply:
I like the repeatable attribute 👍